unit/solvers/smt2_incremental/convert_expr_to_smt.cpp

```diff
@@ -637,3 +637,38 @@ SCENARIO(
     }
   }
 }
+
+SCENARIO(
+  "Bitwise \"NOT\" expressions are converted to SMT terms (1's complement)",
+  "[core][smt2_incremental]")
+{
+  GIVEN("An integer bitvector")
+  {
+    const auto one_bvint = from_integer({1}, signedbv_typet{8});
```

Comment:
Are the `{}` around the `1` surplus to requirements? Can't we just use `from_integer(1, signedbv_typet{8})`?

Reply:
Yes, it is.

I've removed it (Though, it's probably worth mentioning that even if it had stayed, it caused no harm).